src/app/ubs/ubs/components/ubs-confirm-page/ubs-confirm-page.component.html

```diff
@@ -7,7 +7,7 @@
             {{ 'confirm-page.to-personal-account' | translate }}
           </button>
         </div>
-        <div class="error" *ngSwitchCase="orderResponseError === true">
+        <div class="error" *ngSwitchCase="orderResponseError === true || orderPaymentError === true">
```

Comment:
*ngSwitchCase="orderResponseError || orderPaymentError"
please, update this for all switch cases